cli/lib/commands/publish.ts

```diff
@@ -42,6 +42,7 @@ export default runner.create<PublishRunOptions>({
     variable.set('publish-graph-store-password', publishStore?.password || process.env.PUBLISH_GRAPH_STORE_PASSWORD)
     variable.set('metadata', $rdf.dataset())
     variable.set('shapesPath', path.resolve(__dirname, `../../${profile}.ttl`))
+    variable.set('shapesVersion', 'v0.2.2')
```

Comment:
I think I'd move the default directly to the replace step module module and keep the variable only for a potential override from project